src/components/typography/typography.stories.tsx

```diff
@@ -28,7 +28,7 @@ const subData = {
 
 const headers = (
   <div>
-    <H0>H1: Title Enourmous. The five boxing wizards jump quickly.</H0>
+    <H0>H0: Title Enourmous. The five boxing wizards jump quickly.</H0>
```

Comment:
There is no h0 DOM element. Name should be changed I think.

Reply:
So is the case for H1, H2 etc they don't exist. 
In any case propose a name and I can rename it.
Also, this is not related to this PR.